src/Components/Navbar/Navbar.tsx

```diff
@@ -0,0 +1,15 @@
+import './Navbar.css'
+
+function Navbar() {
+  return (
+    <nav className='nav'>
+      <h1>BookWorm</h1>
+      <ul>
+        <li>Home</li>
+        <li>Favorites</li>
+      </ul>
+    </nav>
+  )
+}
+
+export default Navbar
```

Comment:
I've never made a NavBar like this, where list items act as the buttons but I'm excited to try it out and see the pros/cons.